File: Dataset/Leetcode/valid/70/48.py
class Solution:
    def XXX(self, n: int) -> int:
        num = 0
        fibo = [1,1,1]
        for i in range(n):
            if i > 2:
                fibo.append(fibo[i-1]+fibo[i-2])
            num = num + fibo[i]
        return num

File: Dataset/Leetcode/train/12/48.java
 class Solution {
    public String XXX(int num) {
        String[] a1 = {"","M","MM","MMM"};
        String[] a2 = {"","C","CC","CCC","CD","D","DC","DCC","DCCC","CM"};
        String[] a3 = {"","X","XX","XXX","XL","L","LX","LXX","LXXX","XC"};
        String[] a4 = {"","I","II","III","IV","V","VI","VII","VIII","IX"};
        StringBuilder sb = new StringBuilder();
        
        sb.append(a1[num / 1000]);
        sb.append(a2[num % 1000 / 100]);
        sb.append(a3[num % 100 / 10]);
        sb.append(a4[num % 10]);
        
        return sb.toString();
    }
}

